src/tracks/playabletrack/wavetrack/ui/WaveClipUtilities.cpp

```diff
@@ -15,8 +15,12 @@
 #include <cassert>
 #include <cmath>
 
+#include "PitchAndSpeedDialog.h"
+#include "ProjectAudioIO.h"
 #include "ProjectHistory.h"
+#include "ProjectWindows.h"
 #include "UndoManager.h"
+#include "WaveTrack.h"
```

Comment:
As I said, I don't like enlarging this file with the new function, but `#include <wx/frame.h>` would be needed to fix its compilation